docs/source/tutorials/first_kagome_chain.md

```diff
@@ -62,12 +66,21 @@ subkey inspect //Bob
 For this tutorial you can start a single node network as follows:
 
 ```
-kagome_validating \
-    --genesis config/localchain.json \
-    --keystore config/localkeystore.json \
-    --leveldb ldb \
-    --rpc_http_port 40363 \
-    --rpc_ws_port 40364
+docker run \
+  -it \
+  -v $(pwd)/demo:/kagome/configs \
```

Comment:
Hm, is it stated somewhere what's demo/ in the current dir, what is supposed to be there and where should the working dir be?

Reply:
Also, not an issue but just an IMHO statement, config**s** seems a bit inconventional to me, dirs are usually named as single nouns in *nix systems in my experience.